reslib/pom.xml

```diff
@@ -47,10 +47,10 @@
 									<goal>wget</goal>
 								</goals>
 								<configuration>
-									<url>https://github.com/liferay/liferay-portal/raw/0f75db5b5c6a8a7c6b07295cc12acdf362bfb1b4/portal-web/third-party/alloy-${alloyui.zip.version}.zip</url>
+									<url>https://github.com/liferay/liferay-portal/raw/6.2.x/portal-web/third-party/alloy-${alloyui.zip.version}.zip</url>
 									<unpack>true</unpack>
 									<outputDirectory>${project.build.directory}/classes/META-INF/resources</outputDirectory>
-									<md5>37cb9069053f2e181bd26bd78a216c53</md5>
+									<md5>b44a6007099e482c1a5e77fe18018a61</md5>
```

Comment:
@juangon, if we change the branch to `6.2.x`, the build will break the next time they upgrade it due to the md5 not matching. It's much better to specify a commit so the build will always work. That's why I did [this commit](https://github.com/juangon/liferay-faces-alloy/commit/d8831085c59d68827664724f858b7fc43b4e1818).